manifests/init.pp

```diff
@@ -158,6 +179,13 @@
 #
 class consul (
   Hash $acls                                 = $consul::params::acls,
+  Hash $tokens                               = $consul::params::tokens,
```

Comment:
can you enforce the datatypes even more? For hashes, you can define the datatypes for the keys and the values. For example:
```suggestion
  Hash[String[1],String[1]] $tokens          = $consul::params::tokens,
```

Reply:
Good idea. I tried to add some stricter types in the newest version.